### PR TITLE
Minor templating (v2)

### PR DESCRIPTION
**Pull request by SMotaal**
The following changes come from double checking my edits to the tracking sheets.

> I'm still adding to this PR.

Signed-off-by: Saleh Abdel Motaal <saleh@smotaal.io>

### CODE REVIEW

File: focus-areas/ecosystem/social-currency-metric-system.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Social Currency Metric System (SCMS)
+# Ecosystem Value: Social Currency Metric System (SCMS)
```
Comment (GeorgLink) [inline focus-areas/ecosystem/social-currency-metric-system.md]: We have been inconsistent with including the focus area in the title of a metric. Lately, we have removed them.

Can you please remove "Ecosystem Value: " from line 1?

File: focus-areas/ecosystem/social-currency-metric-system.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ Community member comments are available from trace data. The SCMS ideally import
 _Points of destruction_: Detailed data is destroyed after _xx_ months has passed. Quantitative calculations are kept for up to 250 weeks per GDPR compliance. Data older than 250 weeks becomes archived data you cannot manipulate but can see. Users can negotiate the primary statistic.
 
 
-## References
+## Resources
```
Comment (GeorgLink) [inline focus-areas/ecosystem/social-currency-metric-system.md]: Good catch!

Thank you!

File: focus-areas/ecosystem/social-currency-metric-system.md (hunk 1 above)
Comment (SMotaal) [inline focus-areas/ecosystem/social-currency-metric-system.md]: So, I think it helps to use the <code>\`\`\`suggestion</code> fencing like so:

```suggestion
# Social Currency Metric System (SCMS)
```

<small><details>

>     ```suggestion
>     # Social Currency Metric System (SCMS)
>     ```

- It needs to be a seed comment on the line you plan to commit against — ie why I haven't been replying directly to your comments when making `suggestion`s.
- It requires using the UI to make batch or single `suggestion`-commits.
- Not sure how well it pans for multi-line `suggestion`s.
- I use the <kbd>Add single comment</kbd> route, not sure if <kbd>Start a review</kbd> works the same or is reasonably useful.

</details></small>

File: focus-areas/ecosystem/social-currency-metric-system.md (hunk 2 above)
Comment (SMotaal) [reply focus-areas/ecosystem/social-currency-metric-system.md]: I'm leaving this open, I realize I'm more than 15 days in by now and so I'm likely looking for cues (taking in details) so spotting (#not-a-douche) is just me distracting myself (and so thanks for being awesome).

File: focus-areas/ecosystem/social-currency-metric-system.md (hunk 1 above)
Comment (SMotaal) [reply focus-areas/ecosystem/social-currency-metric-system.md]: See below!

File: focus-areas/ecosystem/social-currency-metric-system.md (hunk 2 above)
Comment (GeorgLink) [reply focus-areas/ecosystem/social-currency-metric-system.md]: I checked our template and it is supposed to be "References"

https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md

**Comment by GeorgLink** [comment]
Hi @SMotaal , 

After reviewing our [master template](https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md), it looks like we were already fully compliant. I'm going to close this pr but please do open a new PR if you spot anything else.

File: focus-areas/ecosystem/social-currency-metric-system.md (hunk 2 above)
Comment (SMotaal) [reply focus-areas/ecosystem/social-currency-metric-system.md]: I think I spotted "Resources" in [wg-diversity-inclusion#243](https://github.com/chaoss/wg-diversity-inclusion/pull/243) and likely others that indicated "v2" templates — sorry I did not know/remember checking the template itself — so is it okay then to open a PR to changes "Resources" to "References" (that felt more appropriate in PR I mentioned BTW but wasn't sure it was time to debate stylistic matters there 👍).

Comment (GeorgLink) [reply focus-areas/ecosystem/social-currency-metric-system.md]: Yes, it's appropriate to open PRs to fix any templating mistakes we have in our metrics -- in fact, that would be a huge help.

**Pull request closed without merging**